test/unit/clarity_ext/dilution/hamilton_tests.py

```diff
@@ -17,6 +20,9 @@ class HamiltonTests(unittest.TestCase):
     """
 
     def setUp(self):
+        self.mock = DilutionMock1()
```

Comment:
No! it's not possible to create the mock without FakingEntityMonkey. And, I'm not fond of putting my mock repository of ~100 lines in my unit test class. I dont see the point in 'get rid of several classes' if there is a clear separation of responsibilities. In addition, there is a base class to the DilutionMock1, that I intend to use later to avoid boilerplate code.